### PR TITLE
Koatuu models

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: data_ocean/converter.py [hunk 1]
```diff
@@ -19,6 +19,11 @@ class Converter:
     DOWNLOAD_FOLDER = "download/"  # folder to downloaded files
     URLS_DICT = {}  # control remote dataset files update
 
+
+    def load_json(self, json_file):
+        with open(json_file):
+            return json.load(json_file)
```
Comment (roman-tiukh) [inline data_ocean/converter.py:25]: this work?

File: data_ocean/models.py [hunk 2]
```diff
@@ -8,6 +9,9 @@ class DataOceanModel(models.Model):
     class Meta:
         abstract = True
 
+    def soft_delete(self):
+        self.created_at = now
```
Comment (roman-tiukh) [inline data_ocean/models.py:13]: 1. as far as I remember `auto_now_add` will not allow you to set your own value for the `created_at` field.
2. I think you need to call `now()` for get now value.

File: data_ocean/converter.py (hunk 1 above)
Comment (roman-tiukh) [reply data_ocean/converter.py:25]:
```suggestion
        with open(json_file) as file:
            data = json.load(file)
        return data
```

Comment (IegorSoboliev) [reply data_ocean/converter.py:25]: refactored, thank you!

File: data_ocean/models.py (hunk 2 above)
Comment (IegorSoboliev) [reply data_ocean/models.py:13]: sorry, my fault. I meant another field((

Comment (IegorSoboliev) [reply data_ocean/models.py:13]: fixed

**Pull request merged**